content/03.methods.md

```diff
@@ -5,8 +5,72 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+A common challenge in natural language processing is having too few ground truth annotations, even when data are abundant.
+Data programming [@arxiv:1605.07723] circumvents this issue by quickly annotating large datasets by using multiple noisy signals emitted by label functions.
```

Comment:
I'd place the reference at the end of this sentence because the whole sentence describes data programming. In general references usually go at the end of the setnence, unless additional clarity on what the citation applies to is provided by interspersing them.